ML/MLClassification.py

```diff
@@ -301,20 +301,35 @@ def appl(inputCfg, PtBin, OutPutDirPt, ModelHandl, DataDfPtSel, PromptDfPtSelFor
     print('Applying ML model to data dataframe: Done!')
 
 
-def main(): #pylint: disable=too-many-statements
-    # read config file
-    parser = argparse.ArgumentParser(description='Arguments to pass')
-    parser.add_argument('cfgFileName', metavar='text', default='cfgFileNameML.yml', help='config file name for ml')
-    parser.add_argument("--train", help="perform only training and testing", action="store_true")
-    parser.add_argument("--apply", help="perform only application", action="store_true")
-    args = parser.parse_args()
-
-    print('Loading analysis configuration: ...', end='\r')
-    with open(args.cfgFileName, 'r') as ymlCfgFile:
-        inputCfg = yaml.load(ymlCfgFile, yaml.FullLoader)
-    print('Loading analysis configuration: Done!')
+def main(configFile=None, doTraining=None, doApplication=None): #pylint: disable=too-many-statements
```

Comment:
What is the ratio of doing this?

Reply:
To use this script as a module, is this not ok?

Reply:
Ok, but you are not committing anything that uses this functionality. At the moment, the only modules we have are those in `utils`, everything else is a top-level script. 
If this is not used I would not commit it.